src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1732,7 +1732,7 @@ private TypeDeclarationSyntax ParseClassOrStructOrInterfaceDeclaration(SyntaxLis
             _termState |= TerminatorState.IsEndOfRecordOrClassOrStructOrInterfaceSignature;
 
             var saveTerm = _termState;
-            _termState |= TerminatorState.IsPossibleAggregateClauseStartOrStop;
+            //_termState |= TerminatorState.IsPossibleAggregateClauseStartOrStop; // TODO2 looking for affected test
```

Comment:
mark as prototype?  so this doesn't get merged in?

Reply:
This PR is draft. Not ready for review. TODO2 comments are flagged by our build and are useful to track things that need to be handled in the PR ;-)

Reply:
oh.  gmail didn't mark this as draft.  sorry!

gtk about TODO2.  i didn't realize that.